### PR TITLE
PR for activity Portfolio: accessibility

### PR DESCRIPTION
**Pull request by DarioAlessioR**
This PR is to ask for approval of activity Portfolio: accessibility.

The review was done on branch Accessibility1, for mobile, and for desktop versions.

Accessibility was evaluated according to Microverse instruction, using the following tools:
- WAVE (https://wave.webaim.org/)
- Lighthouse (built-in Chrome feature)
- Nu Html Checker (https://validator.w3.org/nu/)
- Zoom Text Only (with Chrome extension)

The following is the report after the check:

Page titles:--------------------------- no issues found

Image text alternatives:------------- lack of alt text in icons imported images in anchor elements  
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L23
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L82
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L84
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L85
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L86

Text headings:----------------------- 3 text headings in non-hierarchical order
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L46
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L55
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/index.html#L65

Color contrast:---------------------- all buttons with not enough contrast
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/style.css#L236

Resize:------------------------------- text of 4 buttons overflowed its container after resize
https://github.com/DarioAlessioR/my-portfolio/blob/96f658c5f41df8ff71485d111504901171cb2fc8/style.css#L241

Interaction:-------------------------- 3 input fields with no label
https://github.com/DarioAlessioR/my-portfolio/blob/0953210dd648cbc42e63271fdf8fdb494f21adf6/index.html#L208
https://github.com/DarioAlessioR/my-portfolio/blob/0953210dd648cbc42e63271fdf8fdb494f21adf6/index.html#L210
https://github.com/DarioAlessioR/my-portfolio/blob/0953210dd648cbc42e63271fdf8fdb494f21adf6/index.html#L212

Moving content:-------------------- no issue found
Multimedia:------------------------- no issue found
The basic structure of the page:--- no issue found

After performing the check all issues were fixed and the check was done again with no issues found.

Thank you in advance @prolajumokeoni

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
> This PR is to ask for approval of activity Portfolio: accessibility.
> 
> The review was done on branch Accessibility1, for mobile, and for desktop versions.
> 
> Accessibility was evaluated according to Microverse instruction, using the following tools:
> 
> * WAVE (https://wave.webaim.org/)
> * Lighthouse (built-in Chrome feature)
> * Nu Html Checker (https://validator.w3.org/nu/)
> * Zoom Text Only (with Chrome extension)
> 
> The following is the report after the check:
> 
> Page titles:--------------------------- no issues found Image text alternatives:------------- lack of alt text in icons imported images in anchor elements Text headings:----------------------- 3 text headings in non-hierarchical order Color contrast:---------------------- all buttons with not enough contrast Resize:------------------------------- text of 4 buttons overflowed its container after resize Interaction:-------------------------- no issue found Moving content:-------------------- no issue found Multimedia:------------------------- no issue found The basic structure of the page:--- no issue found
> 
> After performing the check all issues were fixed and the check was done again with no issues found.
> 
> Thank you in advance.


For issues found, you should link to the corresponding line of code in the Pull Request changes.

File: style.css [hunk 1]
```diff
@@ -265,7 +264,7 @@ hr {
 }
 
 .superButtonR {
```
Comment (prolajumokeoni) [inline style.css]: - [ ] Kindly change the hover and click effect color for this button in your project as it fails the contrast check 👍🏿 
Screenshot_175

- Please use the color used in the design 👍🏿

File: style.css [hunk 2]
```diff
@@ -363,13 +362,13 @@ article {
 #about {
   color: #9390ff;
   font-weight: 400;
-  font-size: 1.6rem;
+  font-size: 1.3rem;
 }
 
 #contactForm {
```
Comment (prolajumokeoni) [inline style.css]: - [ ] Kindly make sure that every form control has a label associated with it using 'label', 'for', and 'id' in your form,  a [benefit ](https://www.w3schools.com/tags/tag_label.asp) of  using labels is; screen reader users (will read out loud the label, when the user is focused on the element)

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @DarioAlessioR, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design 


## Required Changes ♻️
_Check the comments under the review._

 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
> Hi @DarioAlessioR, Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> ✔️ No linter errors ✔️ Github flow was correctly used ✔️ Good design
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @prolajumokeoni

I have a doubt: 

As pointed out, the project has no labels in the forms. The reason is that Figma template #4 uses just placeholder text and no labels at all.

I understand the benefit of using labels for accessibility. 

So, on behalf of your request and to increase accessibility, labels are to be added to the form.

**Comment by prolajumokeoni** [comment]
Hi @DarioAlessioR making your website accessible to every user is the goal of this milestone, I strongly advise you add them as it is part of the project requirement, good job so far👍

**Review by keroloussamy — APPROVED** [review]
Hi @DarioAlessioR, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Close PR after receive approval for activity Portfolio accessibility

**Pull request merged**